source/scpd/Cpp.d

```diff
@@ -111,6 +111,16 @@ extern(C++, `std`) {
     {
         void*[3] ptr;
     }
+
+    /// Simple bindings to std::chrono
+    public struct chrono
```

Comment:
Isn't `chrono` a namespace ??

Reply:
Right, I just wanted the mangling to match.